bin/blin.p6

```diff
@@ -379,6 +382,27 @@ sub save-overview {
 save-overview;
 
 
+note '🥞🥞 Saving the markdown output';
+sub save-markdown {
+  $save-lock.protect: {
```

Comment:
This is not needed. The idea was that `save-overview` is called multiple times while Blin is still running, so that if the whole thing somehow crashes when testing the full ecosystem you could still take a peek at the results to see what it managed to do so far. For `save-markdown` let's keep it simple.

Reply:
OK, the protect is removed.